packages/api/controllers/nomination.js

```diff
@@ -24,6 +24,19 @@ const getNominationById = async (req, res) => {
   }
 };
 
+const findAllNominataions = async (req, res) => {
+  try {
+    const nominations = await db.Nomination.findAll();
+    if (nominations.length) {
+      return res.status(200).json(nominations);
+    }
+    return res.status(404).send('No nominations found');
+  } catch (error) {
+    console.error('500 on findAllNominations', error);
+    return res.status(500).json({ error: error.message });
```

Comment:
Nit: I think this is fine for debugging, but we probably want to return something generic here (instead of `error.message`). The message is likely to contain technical details about the internal working of the API, which isn't relevant to the frontend.